tests/utils/constants/config.go

```diff
@@ -33,10 +31,10 @@ type RHCSconfig struct {
 func init() {
 	currentDir, _ := os.Getwd()
 	project := "terraform-provider-rhcs"
-	rootDir := GetEnvWithDefault(CON.WorkSpace, strings.SplitAfter(currentDir, project)[0])
+	rootDir := GetEnvWithDefault(WorkSpace, strings.SplitAfter(currentDir, project)[0])
 
 	// defaulted to staging
-	RHCS.RHCSEnv = GetEnvWithDefault(CON.OCMEnv, RHCS.RHCSEnv)
+	RHCS.RHCSEnv = GetEnvWithDefault(RHCSENV, RHCS.RHCSEnv)
 
 	if os.Getenv("CLUSTER_PROFILE") != "" {
 		RHCS.ClusterProfile = os.Getenv("CLUSTER_PROFILE")
```

Comment:
why not using the function `GetEnvWithDefault`?
it set default value in case of null vars

Reply:
os.Getenv("CLUSTER_PROFILE") will give an empty string if no ENV set. So no need to set a default value.

Reply:
CLUSTER_PROFILE is a must set ENV, we should raise error rather that set a default one.